.teamcity/_self/projects/WebApp.kt

```diff
@@ -589,13 +589,50 @@ object Translate : BuildType({
 	name = "Translate"
 	description = "Extract translatable strings from the source code and build POT file"
 
+	artifactRules = """
+		translate => translate
+	""".trimIndent()
+
 	steps {
 		bashNodeScript {
-			name = "Translate Build Steps"
+			name = "Prepare environment"
+			scriptContent = """
+				# Install modules
+				${_self.yarn_install_cmd}
+			"""
```

Comment:
Could you try specifying the docker image for each `bashNodeScript`, like here:

https://github.com/Automattic/wp-calypso/blob/trunk/.teamcity/_self/lib/customBuildType/E2EBuildType.kt#L137